### PR TITLE
change 34 icons

### PR DESCRIPTION
**Pull request by Udobnyj**
Based on this [Size consistency task](https://github.com/mapbox/maki/issues/555), I changed 34 maki icons

### CODE REVIEW

File: layouts/all.json [hunk 1]
```diff
@@ -1 +1 @@
...
...stroked","marker","mobile-phone","monument-JP","monument","mountain","museum","music","natural","observation-tower","optician","paint","park-alt1","park","parking-garage","parking","pharmacy","picnic-site","pitch","place-of-worship","playground","police-JP","police","post-JP","post","prison","racetrack-boat","racetrack-cycling","racetrack-horse","racetrack","rail-light","rail-metro","rail","ranger-station","recycling","religious-buddhist","religious-christian","religious-jewish","religious-muslim","religious-shinto","residential-community","restaurant-bbq","restaurant-noodle","restaurant-pizza","restaurant-seafood","restaurant-sushi","restaurant","road-accident","roadblock","rocket","school-JP","school","scooter","shelter","shoe","shop","skateboard","skiing","slaughterhouse","slipway","snowmobile","soccer","square-stroked","square","stadium","star-stroked","star","suitcase","swimming","table-tennis","teahouse","telephone","tennis","theatre","toilet","toll-booth","town-hall","town","triangle-stroked","triangle","tunnel","veterinary","viewpoint","village","volcano","volleyball","warehouse","waste-basket","watch","water","waterfall","watermill","wetland","wheelchair","windmill","zoo"]
```
Comment (AlexanderBelokon) [inline layouts/all.json:1]: I noticed that `ferry-JP` was renamed to `ferry-jp`, is that intentional? Filename for this icon seems to have stayed the same. Also, other `-JP` are still uppercase.

Comment (Udobnyj) [reply layouts/all.json]: Hey! After the conflict resolving everything was corrected

File: icons/bowling-alley.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 </svg>
```
Comment (willymaps) [inline icons/bowling-alley.svg:4]: Should the finger hole circles continue to be as large as they are in the current design? They seem like important indicators. Also seeing some additional anchors around the finger holes.

File: icons/hot-spring.svg [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+  <path d="M13 8.99995C13.7 11.6163 9.5263 12 7.49997 12C5.47364 12 1.29997 11.5163 1.99997 8.99995C1.36654 9.31626 0.999969 9.96092 0.999969 10.5C0.999969 12.0187 3.91012 13.3999 7.49997 13.3999C11.0898 13.3999 14 12.0187 14 10.5C14 9.96092 13.6334 9.31626 13 8.99995Z"/>
 </svg>
```
Comment (willymaps) [inline icons/hot-spring.svg:5]: ❤️

File: icons/gift.svg [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+<svg width="15" height="15" viewBox="0 0 15 15" xmlns="http://www.w3.org/2000/svg" id="gift">
+  <path d="M10.5 1.00152C9.0967.957116 7.8503 1.89192 7.5 3.25152 7.1497 1.89192 5.9033.957116 4.5 1.00152 2.7678.947006 2.23515 2.79746 2.81831 4H1.5C1.22386 4 1 4.22386 1 4.5V5.5C1 5.77614 1.22386 6 1.5 6H13.5C13.7761 6 14 5.77614 14 5.5V4.5C14 4.22386 13.7761 4 13.5 4H12.1938C12.8 2.81516 12.2922 1.00143 10.5404 1.00112 10.5269 1.00112 10.5135 1.00122 10.5 1.00152ZM10.6831 4H8.5C8.50082 2.8961 9.3959 2.00152 10.5 2.00152 11.675 2.00152 11.8827 3.87414 10.6831 4ZM6.5 4H4.29187C3.25675 3.88256 3.19576 2.10553 4.3323 2.00152 4.3881 1.99642 4.4442 1.99642 4.5 2.00152 5.60409 2.00152 6.49918 2.8961 6.5 4ZM2 7.00146V11.5015C2 12.3299 2.6716 13.0015 3.5 13.0015H6.5V7.00146H2ZM8.5 7.00146V13.0015H11.5C12.3284 13.0015 13 12.3299 13 11.5015V7.00146H8.5Z"/>
 </svg>
```
Comment (willymaps) [inline icons/gift.svg:4]: 💟

**Review by willymaps — COMMENTED** [review]
@Udobnyj such an improvement here! In addition to the comments I made, I also noticed some unnecessary `<g>` tags that were potentially [introduced here](https://github.com/mapbox/maki/pull/565). Can you please update:

- `slipway`
- `slaughterhouse`
- `construction`
- `restaurant-pizza`
- `snowmobile`

File: icons/toilet.svg [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
 </svg>
```
[the remark this answers is not in the document]
Comment (Udobnyj) [reply icons/toilet.svg:4]: Good point! I fixed it ❤️

File: icons/bowling-alley.svg (hunk 2 above)
Comment (Udobnyj) [reply icons/bowling-alley.svg:4]: Thank you, I totally agree. I returned the old holes and deleted unnecessary anchors

**Pull request merged**